index.js

```diff
@@ -0,0 +1,44 @@
+
+
+// First exercise --------------------
+const countMovesUp = inputValue => inputValue.match(/\(/g).length
+
+const countMovesDown = inputValue => inputValue.match(/\)/g).length
+
+const santasFloor = inputValue => countMovesUp(inputValue) - countMovesDown(inputValue)
+
+
+// Second exercise --------------------
+const santasFirstPositionAtSubsolo = ( position, santasFlorPosition) => {
```

Comment:
Acho que ao invés de passar o `position` como argumento, daria pra setar ele aqui mesmo 🤔 dentro do `for`
```js
for (i = 0; i < inputValue.length; i++)
```